components/Global/componentOverride.jsx

```diff
@@ -0,0 +1,103 @@
+export const styleOverrides = {
+  MuiTextField: {
+    defaultProps: {
+      variant: "outlined",
+      InputLabelProps: {
+        shrink: false,
+      },
+    },
+    styleOverrides: {
+      root: ({ theme: { typography, palette, spacing } }) => ({
```

Comment:
move this file inside theme/style-override.js

we prefer to use jsx only when there is `html` inside the file